nixie/unittests/test_verbose.py

```
@@ -30,11 +30,11 @@ async def test_render_block_emits_command(
     semaphore = asyncio.Semaphore(1)
     path = tmp_path / "doc.md"
 
-    async def fake_create_subprocess_exec(*cmd: str, **kwargs: object) -> object:
+    async def fake_create_subprocess_exec(*cmd: str, **_kwargs: object) -> object:
```

Ruff (0.12.2)

Unused function argument: `cmd`

(ARG001)